clang/test/CodeGen/X86/x86-builtins-intrinsic.cl

```diff
@@ -0,0 +1,73 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+// RUN:  %clang_cc1 -no-opaque-pointers %s -triple x86_64-pc-linux  \
```

Comment:
The community has been moving to opaque pointers. In fact, in LLVM 17 (version that is developed now) typed pointers are claimed as not supported and the fixes for them are not accepted.
So, does it make sense to check opaque pointers too?

Reply:
I removed this flag.

Reply:
What I actually meant is to add both versions of the test - opaque and non opaque. This makes sense because community stopped caring about typed pointers, so when we have a typed pointer LIT test this might help to catch the potential problem earlier during pulldown.
Once we switch to opaque pointers in our project too - we already have opaque pointer test and can just remove typed pointer test.